rollup.config.js

```diff
@@ -0,0 +1,18 @@
+// rollup.config.js
+import typescript from "rollup-plugin-typescript2";
+import { terser } from "rollup-plugin-terser";
+
+export default {
+  input: "./src/index.ts",
+  output: {
+    file: "sift.min.js",
+    format: "es"
```

Comment:
it's possible to use rollup to generate all needed version: es5, es5m, es6 and typescript to generate types